src/Header.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import logoCash from './asset/logoCash.png';
+
+const Header = () => (
+  <header className="header">
+    <div className="slogan">
```

Comment:
do you really need the div? it doesn't provide any semantic (like header or h1), so if you don't plan to put css on it, you can just remove it... or if you plan to put css on it, add a basic one to your PR ;)

Reply:
you can let it, it will be usefull in future.